aezeed/cipherseed.go

```diff
@@ -535,6 +536,13 @@ func decipherCipherSeed(cipherSeedBytes [EncipheredCipherSeedSize]byte,
 	if !ok {
 		return plainSeed, salt, ErrInvalidPass
 	}
+	if len(plainSeedBytes) != DecipheredCipherSeedSize {
+		return plainSeed, salt, fmt.Errorf(
+			"unexpected plaintext size: got %d, want %d",
+			len(plainSeedBytes),
+			DecipheredCipherSeedSize,
+		)
```

Comment:
According to the doc (https://github.com/lightningnetwork/lnd/blob/master/docs/code_formatting_rules.md#exception-for-log-and-error-message-formatting), the formatting should be as follows:
```suggestion
		return plainSeed, salt, fmt.Errorf("unexpected plaintext size:"+
			" got %d, want %d", len(plainSeedBytes),
			DecipheredCipherSeedSize)
```